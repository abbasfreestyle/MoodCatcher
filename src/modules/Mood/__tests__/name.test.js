import * as name from '../name';

describe('Modules > Mood > Name', () => {
  test('should match the snapshot', () => {
    expect(name).toMatchSnapshot();
  });
});
