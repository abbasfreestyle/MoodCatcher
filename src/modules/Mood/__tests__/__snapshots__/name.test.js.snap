// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modules > Mood > Name should match the snapshot 1`] = `
Object {
  "NAME": "Mood",
}
`;
